### DISCUSSION TITLE
Is there a migration guide from v1 to v2?

### DISCUSSION THREAD

**Discussion by joeprivettsmart** [json5/json5#312] [Q&A] [answered]
Is there a migration guide from v1 to v2?

**Comment by jordanbtucker** [comment, marked as answer]
There's no migration guide for v2, but the only breaking change is that Node.js v6 is the oldest version of Node.js supported.

For more details see [v2.0.0 in the CHANGELOG](https://github.com/json5/json5/blob/main/CHANGELOG.md#v200-code-diff).
